libsplinter/Cargo.toml

```diff
@@ -105,7 +105,6 @@ experimental = [
     # The following features are experimental:
     "admin-service-client",
     "admin-service-event-store",
```

Comment:
Did you intend to also remove admin-service-event-store?

Reply:
Yes, I tried to separate removing the two features and forgot this line when I updated the PR - fixed now